swagger.yml

```diff
@@ -903,12 +901,16 @@ components:
           properties:
             dbName:
               type: string
+              description: The name of the database
             hostname:
               type: string
+              description: The name of the host
         hostname:
           type: string
+          description: The name of the host
         description:
           type: string
+          description: 'The message alert receive from the host at that time '
```

Comment:
At  this moment agent don't generate and send alerts. All alerts are generated at backend side

Reply:
This description is about the alert generated by the host at an given time and not by the agent. I do not see nothing wrong with this description.

Reply:
Ok